### DISCUSSION TITLE
Add task list items to a table

### DISCUSSION THREAD

**Discussion by mikelkew** [community/community#5183] [Projects and Issues]
Hi there, it'd be super handy to be able to add task list items to a table.

Currently you need to use emojis/text to indicate status. For example:

#### ENV vars to add:

| ENV var | Staging | Production |
| :--- | :---: | :---: |
| `VAR1` | ✅ | ❌ |
| `VAR2` | ✅ | ❌ |

What I'd like to be able to do (and have the checkboxes toggle-able):

```
| ENV var | Staging | Production |
| :--- | :---: | :---: |
| `VAR1` | - [x] | - [ ] |
| `VAR2` | - [x] | - [ ] |
```

To give something similar to this (but toggle-able):

| ENV var | Staging | Production |
| :--- | :---: | :---: |
| `VAR1` | <ul><li>- [x] </li></ul> | <ul><li>- [ ] </li></ul> |
| `VAR2` | <ul><li>- [x] </li></ul> | <ul><li>- [ ] </li></ul> |

**Comment by rossjrw** [comment]
I've also wanted this feature in the past - I had a matrix of tasks to complete and it would have been handy to be able to check all of them off one by one.

**Comment by HonkingGoose** [comment]
I think this is a duplicate of https://github.com/github/feedback/discussions/4680
